src/test/utils.tsx

```diff
@@ -82,3 +88,22 @@ export const hasToast = async (
   expect(card).toBeInTheDocument();
   expect(card).toHaveAttribute("data-type", severity);
 };
+
+export const hasEmptyState = async (
+  title: string,
+  message?: string,
+  buttonLabel?: string,
+) => {
+  const emptyStateHeader = await screen.findByRole("heading", { name: title });
+  expect(emptyStateHeader).toHaveClass("p-heading--4");
```

Comment:
I don't think we need to check the class name as that is an implementation detail of the empty state component.